src/rdsn/include/dsn/dist/replication/replication_enums.h

```diff
@@ -157,4 +157,14 @@ ENUM_REG(replication::manual_compaction_status::QUEUING)
 ENUM_REG(replication::manual_compaction_status::RUNNING)
 ENUM_REG(replication::manual_compaction_status::FINISHED)
 ENUM_END2(replication::manual_compaction_status::type, manual_compaction_status)
+
+ENUM_BEGIN2(replication::backup_status::type, backup_status, replication::backup_status::INVALID)
+ENUM_REG(replication::backup_status::INVALID)
+ENUM_REG(replication::backup_status::CHECKPOINTING)
+ENUM_REG(replication::backup_status::CHECKPOINTED)
+ENUM_REG(replication::backup_status::UPLOADING)
+ENUM_REG(replication::backup_status::SUCCEED)
+ENUM_REG(replication::backup_status::FAILED)
+ENUM_REG(replication::backup_status::CANCELED)
```

Comment:
typo: CANCELLED

Reply:
Well, canceled is the past tense of cancel, not cancelled.